kubernetes/assets/monitors/monitor_unavailable_pods.json

```diff
@@ -1,28 +0,0 @@
-{
-	"name": "[kubernetes] Monitor Kubernetes Unavailable Pods in Deployments",
```

Comment:
Trying to remember why we decided to drop this. Do you know? This sounds useful in general

Reply:
I believe because this scenario was covered by one of the other monitors. If that's not the case, then I can add it back in in a separate review.